node/utils.go

```diff
@@ -310,13 +311,22 @@ func createVolumeDir(hostPath, containerSpec, imageSpec, userSpec, permissionSpe
 	createVolumeDirMutex.Lock()
```

Comment:
it's still very tempting to try the hostpath based map of locks since we'll probably get to under 45 seconds of initial startup for everything in a mostly safe manner